ngraph/core/include/ngraph/op/non_max_suppression.hpp

```diff
@@ -375,9 +375,7 @@ namespace ngraph
                 using Node::set_output_type;
 
                 int64_t max_boxes_output_from_input() const;
-                float iou_threshold_from_input() const;
-                float score_threshold_from_input() const;
-                float soft_nms_sigma_from_input() const;
```

Comment:
In general this change affects the backward compatibility, because it is a public API. In order to avoid issues with backward compatibility we can introduce the new version of operation.

Reply:
I have restored these functions, thanks.